apps/web/app/api/getAllBookmarks/route.ts

```diff
@@ -0,0 +1,33 @@
+import prisma from "@repo/db/client";
+
+export async function POST(req: Request) {
+  const { userid} = await req.json();
```

Comment:
bad
userId shouldnt come from the frontend.
It should be decoded from the `session` variable
Check https://github.com/code100x/daily-code/blob/main/apps/web/app/api/code/submission/route.ts#L20